items/src/main/scala/weco/api/items/services/SierraItemUpdater.scala

```diff
@@ -62,6 +72,45 @@ class SierraItemUpdater(sierraService: SierraService)(
           .getOrElse(item)
     } toSeq
 
+  private def getAccessCondition(itemData: SierraItemData): AccessCondition = {
+    // The bib ID is used for debugging purposes; the bib status is only used
+    // for consistency checking. We can use placeholder data here.
+    val (accessCondition, _) = SierraItemAccess(
+      bibId = SierraBibNumber("0000000"),
+      bibStatus = None,
+      location = itemData.fixedFields
+        .get("79")
+        .flatMap(_.display)
+        .flatMap(
+          name => SierraPhysicalLocationType.fromName(itemData.id, name)
+        ),
+      itemData = itemData
+    )
+
+    accessCondition
+  }
+
+  def getAccessConditions(
+    itemNumbers: Seq[SierraItemNumber]
+  ): Future[Map[SierraItemNumber, AccessCondition]] = {
+    for {
+      itemEither <- sierraSource.lookupItemEntries(itemNumbers)
+
+      accessConditions = itemEither match {
+        case Right(SierraItemDataEntries(_, _, entries)) =>
+          entries.map(item => item.id -> getAccessCondition(item)).toMap
+        case Left(
+            SierraItemLookupError.MissingItems(missingItems, itemsReturned)
+            ) =>
+          warn(s"Item lookup missing items: ${missingItems}")
```

Comment:
nit

```suggestion
          warn(s"Item lookup missing items: $missingItems")
```

Reply:
I think it's time to add this config to `scalafmt` 🙂 !
```
rewrite.rules = [SortImports, RedundantBraces]
rewrite.redundantBraces.stringInterpolation = true
```